lnrpc/lightning.proto

```diff
@@ -746,6 +746,10 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    // A list of transaction hashes. When using REST, this field must be
```

Comment:
Unfortunately the REST library we use doesn't support specifying slices/arrays of parameters for `GET` requests. Maybe we should add an additional `POST` mapping (e.g. `/v1/transactions/list`) [here](https://github.com/lightningnetwork/lnd/blob/c26917ee1fcf7975a0d58a2ebb02f438217dd3f6/lnrpc/lightning.yaml#L14)? Then this comment would need to specify that when using REST, the `POST` variant of this call needs to be used in order for this parameter to work.

Reply:
I'll keep it an GET endpoint for now.